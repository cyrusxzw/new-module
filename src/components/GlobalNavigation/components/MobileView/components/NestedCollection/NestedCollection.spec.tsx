// import React from 'react';
// import { render, screen } from '@testing-library/react';
// import { NestedCollection } from './NestedCollection';
// import { NestedCollectionFixture } from './NestedCollection.fixture';

describe('<GlobalNavigation.MobileView.NestedCollection />', () => {
  it.todo('should assert something');
});
