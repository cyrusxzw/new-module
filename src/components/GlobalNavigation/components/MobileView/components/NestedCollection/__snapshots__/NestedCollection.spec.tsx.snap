// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GlobalNavigation.MobileView.NestedCollection /> should render base component correctly 1`] = `
<div>
  <button
    aria-expanded="true"
    aria-haspopup="true"
    aria-hidden="false"
    class="base dark itemElement nestedButton"
    tabindex="-1"
    title="Nested Collection"
    type="button"
  >
    Nested Collection
    <span
      class="icon expanded dark"
    />
  </button>
  <ul
    aria-hidden="false"
    aria-label="submenu"
    class="list sublist collapse"
    style="max-height: 0px;"
  />
  ,
</div>
`;
